app/auth/layout.tsx

```diff
@@ -1,5 +1,23 @@
-const AtuthLayout = ({ children }: { children: React.ReactNode }) => {
-  return <div className=''>{children}</div>;
+import { Providers } from '@/components/providers/redux-providers';
```

Comment:
You should not change this layout, so keep the previous change.

Reply:
Done